notebooks/Understand_Tables.ipynb

```diff
@@ -0,0 +1,4387 @@
+{
```

Comment:
<p>"occurred under under" -&gt; "occurred under"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Fixed now.